src/test/java/uk/gov/hmcts/dts/fact/services/CourtServiceTest.java

```diff
@@ -125,6 +126,35 @@ void shouldReturnListOfTenCourts() {
         assertThat(results.get(0)).isInstanceOf(CourtWithDistance.class);
     }
 
+    @Test
+    void shouldReturnListOfCourtsWithRelevantLocalAuthority() {
+        final MapitData mapitData = mock(MapitData.class);
+        when(mapitData.getLat()).thenReturn(LAT);
+        when(mapitData.getLon()).thenReturn(LON);
+        when(mapitData.getLocalAuthority()).thenReturn(Optional.of(LOCAL_AUTHORITY_NAME));
+        when(mapitService.getMapitData(JE2_4BA)).thenReturn(Optional.of(mapitData));
+
+        final List<uk.gov.hmcts.dts.fact.entity.CourtWithDistance> courts = new ArrayList<>();
+        for (int i = 0; i < 10; i++) {
+            final uk.gov.hmcts.dts.fact.entity.CourtWithDistance mock = mock(uk.gov.hmcts.dts.fact.entity.CourtWithDistance.class);
+            final AreaOfLaw areaOfLaw = new AreaOfLaw();
+            areaOfLaw.setName(AREA_OF_LAW_NAME);
+            final List<AreaOfLaw> areasOfLaw = singletonList(areaOfLaw);
+            when(mock.getAreasOfLaw()).thenReturn(areasOfLaw);
+            courts.add(mock);
+        }
+        when(courtWithDistanceRepository.findNearestTenByAreaOfLawAndLocalAuthority(anyDouble(), anyDouble(), anyString(), anyString())).thenReturn(
```

Comment:
Within this test, you have set up the lat/long, area of Law and local authority name. If this is the values you are expecting, normally we will pass in those values in the mock. There is nothing wrong with using anyXX() but the convention is use the specific value if that is what in your test :)